src/markdown_generators.js

```diff
@@ -260,35 +191,98 @@ const generateCourseSectionMarkdown = (page, courseData, pathLookup) => {
     */
   try {
     return `${formatHTMLMarkDown(
-      page,
+      page["text"],
       courseData,
-      "text",
       pathLookup
     )}${generateCourseFeaturesMarkdown(
       page,
       courseData,
       pathLookup
-    )}${formatHTMLMarkDown(page, courseData, "bottomtext", pathLookup)}`
+    )}${formatHTMLMarkDown(page["bottomtext"], courseData, pathLookup)}`
   } catch (err) {
     loggers.fileLogger.error(err)
     return page["text"]
   }
 }
 
-const generatePdfMarkdown = (file, courseData) => {
+const getResourceType = mimeType => {
+  switch (mimeType) {
+  case "application/pdf":
+    return RESOURCE_TYPE_DOCUMENT
+  case "image/gif":
+  case "image/jpeg":
+  case "image/png":
+  case "image/svg+xml":
+  case "image/tiff":
+    return RESOURCE_TYPE_IMAGE
+  default:
+    return RESOURCE_TYPE_OTHER
+  }
+}
+
+const generateResourceMarkdownForFile = file => {
   /**
   Generate the front matter metadata for a PDF file
   */
-  const pdfFrontMatter = {
+  const frontMatter = {
     title:         file["title"],
     description:   file["description"],
-    layout:        "pdf",
     uid:           file["uid"],
-    parent_uid:    file["parent_uid"],
+    resourcetype:  getResourceType(file["file_type"]),
     file_type:     file["file_type"],
-    file_location: helpers.stripS3(file["file_location"])
+    file_location: helpers.stripS3(file["file_location"]),
+    layout:        "resource"
```

Comment:
Still seeing `layout: "resource"` here.  Can we remove it here too?

Reply:
Sure, should be gone now